environment.yml

```diff
@@ -1,26 +1,34 @@
 # You can use this file to create a conda environment for this pipeline:
 #   conda env create -f environment.yml
-name: nf-core-nanoseq-1.0.0
+name: nf-core-nanoseq-1.1.0dev
 channels:
   - conda-forge
   - bioconda
   - defaults
 dependencies:
-  - conda-forge::python=3.7.3
-  - conda-forge::markdown=3.1.1
-  - conda-forge::pymdown-extensions=6.0
-  - conda-forge::pygments=2.5.2
+  - conda-forge::python=3.8.6
+  - conda-forge::markdown=3.3.3
+  - conda-forge::pymdown-extensions=8.0.1
+  - conda-forge::pygments=2.7.2
   - conda-forge::pigz=2.3.4
-  - conda-forge::psutil=5.7.0
-
+  - conda-forge::seaborn=0.10.1
+  - conda-forge::r-base=4.0.3
+  
   - bioconda::fastqc=0.11.9
-  - bioconda::multiqc=1.8
+  - bioconda::multiqc=1.9
   - bioconda::qcat=1.1.0
-  - bioconda::pycoqc=2.5.0.3
-  - bioconda::nanoplot=1.28.4
-  - bioconda::samtools=1.9
+  - bioconda::nanoplot=1.32.1
+  - bioconda::samtools=1.11
   - bioconda::minimap2=2.17
   - bioconda::graphmap=0.6.3
   - bioconda::bedtools=2.29.2
-  - bioconda::ucsc-bedgraphtobigwig=357
-  - bioconda::ucsc-bedtobigbed=357
+  - bioconda::ucsc-bedgraphtobigwig=377
+  - bioconda::ucsc-bedtobigbed=377
+  - bioconda::stringtie=2.1.4
+  - bioconda::subread=2.0.1
+  - bioconda::bioconductor-bambu=1.0.0
+  - bioconda::bioconductor-deseq2=1.30.0
+  - bioconda::bioconductor-drimseq=1.18.0
+  - bioconda::bioconductor-dexseq=1.36.0
+  - bioconda::bioconductor-stager=1.12.0
+  - bioconda::bioconductor-bsgenome=1.58.0
```

Comment:
```suggestion
  - bioconda::bioconductor-bsgenome=1.58.0
  
```

Reply:
Fixed in https://github.com/nf-core/nanoseq/pull/89/commits/9089ec58a0c76652c468d6ff6957346e9135039a